build-complete-updater/updater.py

```diff
@@ -0,0 +1,52 @@
+from kubernetes import client, config
+import os
+import time
+import requests
+import json
+
+NAMESPACE = "broker"
+BROKER_URL = "http://broker-service.broker"
+BROKER_PORT = "8080"
+SLEEP_INTERVAL = 5
+BUILD_ID_LENGTH = 6
```

Comment:
Just to future proof this a bit, this parameter should be able to be configurable with environment variables.